centraldogma/src/main/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplier.java

```diff
@@ -123,14 +123,14 @@ public <T> Optional<Property<T>> getProperty(PropertyDefinition<T> definition) {
         child.watch(node -> {
             try {
                 setValue(prop, node);
-            } catch (RuntimeException e) {
+            } catch (Exception e) {
```

Comment:
Maybe it's better to leave a short comment, or it could be reverted back to RE by "code cleaners"...